api/swagger.yaml

```diff
@@ -8585,6 +8585,7 @@ paths:
             - `label=<engine label>`
             - `membership=`(`accepted`|`pending`)`
             - `name=<node name>`
+            - `node-label=<node label>`
```

Comment:
also thinking if this should be `node.label` (full stop, instead of hyphen), as was proposed in https://github.com/moby/moby/issues/28209

ping @vdemeester @dperny?